services/director-v2/src/simcore_service_director_v2/api/routes/computations.py

```diff
@@ -120,25 +121,26 @@ async def create_computation(
                 status_code=status.HTTP_400_BAD_REQUEST,
                 detail=f"Project {job.project_id} is not a valid directed acyclic graph!",
             )
-
-        if job.start_pipeline:
-            # find the entrypoints, if not the pipeline cannot be started
-            entrypoints = find_entrypoints(dag_graph)
-            if not entrypoints:
-                raise HTTPException(
-                    status_code=status.HTTP_422_UNPROCESSABLE_ENTITY,
-                    detail=f"Project {job.project_id} has no services to compute",
-                )
+        # get a subgraph if needed
+        if job.subgraph:
+            dag_graph = create_minimal_graph_based_on_selection(dag_graph, job.subgraph)
```

Comment:
So basically here is the key of the new feature. You build a subgraph from user's node selection